project/Dependencies.scala

```diff
@@ -5,7 +5,8 @@ object Dependencies {
   object versions {
     val scala212 = "2.12.7"
     val scala211 = "2.11.12"
-    val crossScala = Seq(scala211, scala212)
+//    val crossScala = Seq(scala211, scala212)
+    val crossScala = Seq(scala212)
```

Comment:
I think you can remove this, right? It's not used anywhere anymore. If you also update the travis cross-versions I will merge this